README.md

```diff
@@ -35,15 +35,35 @@ brew update
 brew upgrade fzf-make
 ```
 
-## Arch Linux
+### nix
```

Comment:
Here is the installation section.
I want you to write how to develop using nix [here](https://github.com/kyu08/fzf-make/blob/0217175558ac105e5cf048085354c34d14eded6d/README.md#L55)(between `# 💡 Usage` and `# 👥 Contribution`) starting by `# 💻 Development`.

Apart from that, I would like you to describe how to install fzf-make on nixOS in the installation section in the following format.

```md
# 📦 Installation
...

## Arch Linux
...

## NixOS
<!-- please write how to install to NixOS here! -->
...

## OS-independent method
...
```

If you have any questions, feel free to ask me. 😄